src/Arcus.WebApi.Jobs/KeyVault/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Arcus.Security.Core.Caching;
+using GuardNet;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Arcus.WebApi.Jobs.KeyVault
+{
+    /// <summary>
+    /// Extensions on the <see cref="IServiceCollection"/> to make the registration of jobs more dev-friendly.
+    /// </summary>
+    // ReSharper disable once InconsistentNaming
+    public static class IServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Adds a background job to the <see cref="IServiceCollection"/> to automatically invalidate cached Azure Key Vault secrets.
+        /// </summary>
+        /// <param name="services">The services collection to add the job to.</param>
+        /// <param name="cachedSecretProvider">The cached secret provider where the secrets should be invalidated.</param>
+        /// <param name="subscriptionName">The name of the Azure Service Bus subscription that will be created to receive Key Vault events.</param>
+        /// <param name="serviceBusTopicConnectionStringConfigKey">The configuration key that points to the Azure Service Bus Topic connection string.</param>
+        public static IServiceCollection AddAutoInvalidateKeyVaultSecretBackgroundJob(
+            this IServiceCollection services,
+            ICachedSecretProvider cachedSecretProvider,
```

Comment:
Isn't this something we should just get via DI? Keep in mind that this should be registered by them, not us.

Reply:
Yep, removed that.